### DISCUSSION TITLE
IgxQueryBuilderModule cannot be loaded

### DISCUSSION THREAD

**Discussion by danielecrespimininf** [IgniteUI/igniteui-angular#13006] [Q&A] [answered]
Dear everyone, 

I'm trying to use IgxQueryBuilder, but it seems IgxQueryBuilderModule cannot be resolved. 
I tried importing the module via:

- import { IgxQueryBuilderModule } from "igniteui-angular";
- import { IgxQueryBuilderModule } from "@infragistics/igniteui-angular";

Both options lead me to the  **Module has no exported member 'IgxQueryBuilderModule'.**
Current dependencies: "@infragistics/igniteui-angular": "^14.1.1" & "igniteui-angular": "^14.0.0"
What am I doing wrong? I can't find any related question or hint. Also, I have already succesfully imported IgxGridModule in the same solution, so it doesn't seem to be a package-related issue.

Any help would be appreciated.
Thanks in advance, 

Daniele

**Comment by ddaribo** [comment, marked as answer]
Hi Daniele,

The `IgxQueryBuilder` was extracted as a separate component in version `14.2.0`. More information can be found in our [Changelog](https://github.com/IgniteUI/igniteui-angular/blob/master/CHANGELOG.md#1420).

Although the caret symbol “^” on the "@infragistics/igniteui-angular" dependency version should mean it targets the highest available minor and patch versions, my suggestion is to make sure the version is indeed 14.2.x. You could check the package-lock.json file.
 
If the issue persists, please, provide a sample reproducing it, so that we can investigate this further. Currently, the demos in our documentation for the [Query Builder topic](https://www.infragistics.com/products/ignite-ui-angular/angular/components/query-builder) work as expected.
